index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon Marketing Acessability website</title>
+  </head>
+
+  <!-- header section with nav bar containing links -->
+  <body>
+    <!-- nav - header tag-->
+    <header class="header">
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <!-- navigation bar with the website links -->
+      <nav>
+        <ul>
+          <li>
+            <a href="#search-engine-optimization">Search Engine Optimization</a>
+          </li>
+          <li>
+            <a href="#online-reputation-management">Online Reputation Management</a>
+          </li>
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </nav>
+  </header>
+    <!-- nav tag -->
+    <aside class="hero"></aside>
+    <aside class="content"></aside>>
```

Comment:
Please review this as you have a syntax error (an extra `>`) and also - please review the necessity of this `<aside>` as it doesn't contain anything....